release/README.txt

```diff
@@ -1,74 +0,0 @@
-This is the build directory for binary releases, such as Windows and Mac OSX.
-
-Each subdirectory is a git submodule where each commit is a release, tagged so,
-ready to be zipped. The zipfile for each release can be automatically generated
-and downloaded from github. For example, for the Windows release v0.30c the URL:
-
-    https://github.com/singularity/singularity-windows/archive/v0.30c.zip
-
-Will trigger the download of a file named `singularity-windows-0.30c.zip`.
-
-If you are a developer and want to create and publish a new release, first
-fetch and update its submodules. From the singularity root directory, run:
-
-git submodule update --init release/windows
-
----
```

Comment:
Those "release steps" should still be present somewhere. They likely need updates, but are around 90% accurate and a useful checklist for releases.